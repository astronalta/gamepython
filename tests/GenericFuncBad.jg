

badret(bob :a) :c {
    ret nil
}


BadRet[:a] < Object {

    badret() :d {
        ret nil
    }
}
