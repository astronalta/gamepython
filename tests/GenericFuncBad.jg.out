GenericFuncBad.jg:3:16: Unbound generic type ':c'
GenericFuncBad.jg:10:14: Unbound generic type ':d'
